src/test/java/uk/gov/hmcts/reform/civil/service/PaymentsServiceTest.java

```diff
@@ -64,92 +73,113 @@ class PaymentsServiceTest {
     @MockBean
     private FeatureToggleService featureToggleService;
 
-    @Nested
-    class Unspecified {
-        @BeforeEach
-        void setUp() {
-            given(paymentsClient.createCreditAccountPayment(any(), any())).willReturn(PAYMENT_DTO);
-            given(paymentsConfiguration.getService()).willReturn(SERVICE);
-            given(paymentsConfiguration.getSiteId()).willReturn(SITE_ID);
-            given(organisationService.findOrganisationById(any())).willReturn(Optional.of(ORGANISATION));
-        }
-
-        @Test
-        void shouldCreateCreditAccountPayment_whenValidCaseDetails() {
-            uk.gov.hmcts.reform.ccd.model.Organisation orgId = uk.gov.hmcts.reform.ccd.model.Organisation.builder()
-                .organisationID("OrgId").build();
+    @BeforeEach
+    void setUp() {
+        given(paymentsClient.createServiceRequest(any(), any())).willReturn(PAYMENT_SERVICE_RESPONSE);
+        given(paymentsConfiguration.getService()).willReturn(SERVICE);
+        given(paymentsConfiguration.getSiteId()).willReturn(SITE_ID);
+        given(paymentsConfiguration.getSpecSiteId()).willReturn(SPEC_SITE_ID);
+        given(paymentsClient.createPbaPayment(any(), any(), any())).willReturn(PAYMENT_DTO);
+        given(organisationService.findOrganisationById(any())).willReturn(Optional.of(ORGANISATION));
+    }
 
-            CaseData caseData = CaseDataBuilder.builder().atStateClaimSubmitted()
-                .applicant1OrganisationPolicy(OrganisationPolicy.builder().organisation(orgId).build())
-                .build();
+    @Test
+    void validateRequestShouldNotThrowAnError_whenValidCaseDataIsProvided() {
+        CaseData caseData = CaseDataBuilder.builder().buildMakePaymentsCaseData();
```

Comment:
Let's put `// Given`, `// When`, `// Then` comments in these tests.

See PaymentRequestUpdateCallbackServiceTest for bare minimum examples. Better if besides the comment there is a description. Very useful in particular for `// Then`.

This applies to all the tests lacking those comments in this PR.